packages/database/supabase/schemas/embedding.sql

```diff
@@ -30,40 +30,6 @@ GRANT ALL ON TABLE public."ContentEmbedding_openai_text_embedding_3_small_1536"
 
 set search_path to public, extensions ;
 
-CREATE OR REPLACE FUNCTION public.match_content_embeddings (
```

Comment:
I don't know if this function should be removed.
On the one hand you are correct that it is not currently used. OTOH It allows functionality that is not there otherwise, and _could_ be used.
That said... Search architecture will be rethought soon, so it's not like we're going to get to use it in this exact shape.
I guess I'm a -0 on this. Not going to fight it.

Reply:
I mis-remembered and thought I created both these functions and now the other one is being used, I don't have any opinions on which one to keep. Since you see a use for this in future I will keep this one and remove the other one